### PR TITLE
Feat- timer for POMODORO

### PR DESCRIPTION
**Pull request by abhij1607**
- added timer functionality with pause/reset and pomodoro, short break and long break
- updated styles of button
- increased font-size of task in TaskPane

### CODE REVIEW

File: src/components/pomodoro-pane.js [hunk 1]
```diff
@@ -1,15 +1,114 @@
...
+  const [timer, setTimer] = useState(1500);
+  const [pause, setPause] = useState(true);
+  const [menuOptionActive, setMenuOptionActive] = useState("Pomodoro");
+
+  const tick = () => {
+    setTimer(timer - 1);
+  };
+
+  useEffect(() => {
+    var id;
```
Comment (shivampandey0) [inline src/components/pomodoro-pane.js]: use `let` here

**Review by shivampandey0 — COMMENTED** [review]
Good coding practices followed and easy-to-follow code.

- Added a suggestion

File: src/components/pomodoro-pane.js (hunk 1 above)
Comment (abhij1607) [reply src/components/pomodoro-pane.js]: done, thanks for suggestion

**Review by Hamdrive — COMMENTED** [review]
Nice implementation of timer, especially liked the use of switch statement to handle time and breaks.
Overall logic applied is good, deployment works and can be merged

**Pull request merged**